conf/hstream.yaml

```diff
@@ -31,6 +30,12 @@ hserver:
   #
   #compression: "lz4"
 
+  # The max record payload size the hstream server would accept. The value only indicates that the server would not
+  # reject the value, but has no guarantee that a record smaller than the specified value will be written successfully.
+  # Optional; If not specified, the default value will be 1MB
+  #
+  #max-record-size: 1024*1024
```

Comment:
yaml doesn't support arithmetic operation, instead, you can write as the following

```
   #max-record-size: 1048576   # 1024*1024
```